app/assets/javascripts/_common.js

```diff
@@ -62,6 +62,32 @@ $(document).ready(function() {
      $('#filter_container').toggle('fast');
    });
 
+  if ($('#Use_Bulk_Note')) {
+    var bulk_note_box = $('#Use_Bulk_Note');
+    var bulk_place = $('#bulk-note-place');
+    bulk_place.hide()
+    var row_notes = $('.single-note-place')
```

Comment:
I'd prefer using explicit/descriptive names, double quotes, and camelCase :
```suggestion
    var useBulkNoteCheckbox = $("#use-bulk-note");
    var bulkNoteInput = $("#bulk-note-input");
    bulkNoteInput.hide()
    var rowNoteInputs = $(".row-note-input")
```